frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+
+interface Ibutton {
+  btnType?: "primary" | "secondary" | "simple";
+  type?: "button" | "submit";
+  onClick?: any;
```

Comment:
This type should be `onClick?: () => void`
Which means: The type is a function which returns nothing

Reply:
oh man, totally right, since i had submit functions on the buttons instead of the forms, this originally gave all kinds of errors but now that they are moved to the correct place its working